### PR TITLE
add tests for knplabs support

### PR DESCRIPTION
**Pull request by almacbe**
I am targeting this branch, because it adds tests.

It is related to the last comment of #117

### CODE REVIEW

**Comment by almacbe** [comment]
Could @dbu or @Soullivaneuh give me some feedback about this PR if I am in the right direction?

File: Tests/AdminExtension/Knplabs/TranslatableAdminExtensionTest.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+/**
+ * @group  translatable-knplabs
+ */
+class TranslatableAdminExtensionTest extends WebTestCase
+{
+    /**
+     * @var AdminInterface
+     */
+    protected $admin;
```
Comment (dbu) [inline Tests/AdminExtension/Knplabs/TranslatableAdminExtensionTest.php]: please use private unless there is an explicit reason to share something with extending classes.

Comment (core23) [reply Tests/AdminExtension/Knplabs/TranslatableAdminExtensionTest.php]: I think we can live with that in the tests

Comment (dbu) [reply Tests/AdminExtension/Knplabs/TranslatableAdminExtensionTest.php]: agreed that its no big deal. but code quality matters in tests too, and its also no big deal to fix it.

**Comment by OskarStark** [comment]
Thank you @almacbe 👍

**Comment by almacbe** [comment]
Thanks!

**Pull request merged**